### ISSUE TITLE
[Bug] New NiceHashMiner 3.1.0.3 is missing 2 algorithms on GMiner

### ISSUE THREAD

**Issue by MolochBaal** [nicehash/NiceHashMiner#2820]
As the title says, the PluginSupportedAlgorithmsSettings.json is missing 

>      {
>        "IDs": "CuckooCycle",
>        "Enabled": null,
>        "ExtraLaunchParameters": null,
>        "NonDefaultRAMLimit": null
>      },
>      {
>        "IDs": "GrinCuckatoo32",
>        "Enabled": null,
>        "ExtraLaunchParameters": null,
>        "NonDefaultRAMLimit": null
>      },

And since Aeternity is one of the most profitable coins, this is critical 

Also lolMiner keep showing GrinCuckatoo31 on the Extra Commands tab, wich is obsolete and unsupported (should be removed in the same file, but for lolMiner folder)

Another bug: the text of the notifications is black, so it's very hard to read in dark mode.

**Comment by object05** [comment]
Hi, 
CuckooCycle and GrinCuckatoo32 have been intentionally left out at this time because there is a problem causing frequent restaring while using these two algorithms (in NiceHashMiner). This is not permanent, so the algos will probably be re-added soon. Also thank you for reporting the text color bug.

**Comment by MolochBaal** [comment]
> Hi, CuckooCycle and GrinCuckatoo32 have been intentionally left out at this time because there is a problem causing frequent restaring while using these two algorithms (in NiceHashMiner). This is not permanent, so the algos will probably be re-added soon. Also thank you for reporting the text color bug.

Hi, thanks for the reply. I think I had the restart miner error with CuckooCycle, it was caused by the 5 minutes time set in MinerApiMaxTimeoutSetting.json so I disabled it with

>  "use_user_settings": true,
>  "enabled": false,
>  "general_timeout": "24:00:00",

And I have a weird one that blue screen/reboot my computer as soon as it launch CuckooCycle in NiceHashMiner, but only in my mining rig, so yeah, it's better to keep it disabled (also happened in 3.1.0.2).

But no error, that I know, mining Grin.

--

Also would be great to make control groups for the cards, like what MSI Afterburner does, just editing 1 gpu would apply the changes for every similar card (very useful for testing single algorithms).
